core/src/test/java/kafka/test/ClusterInstance.java

```diff
@@ -188,15 +188,15 @@ default Set<GroupProtocol> supportedGroupProtocols() {
     default void waitForTopic(String topic, int partitions) throws InterruptedException {
         // wait for metadata
         TestUtils.waitForCondition(
-            () -> brokers().values().stream().allMatch(broker -> partitions == 0 ?
+            () -> aliveBrokers().values().stream().allMatch(broker -> partitions == 0 ?
```

Comment:
If we don't have this modification, isolated-raft always failed.
Maybe it is waiting the shutdown broker?